### PR TITLE
br.m3u Updates

### PR DESCRIPTION
**Pull request by rblopes**
Hi. First time contributor here, with some updates to the `br.m3u` list. More to come.

- Removed "TV MCJ", as this channel is not a Brazilian TV broadcast.
- Removed a few "TV Aparecida" duplicate entries. Those entries used a IP address instead of the actual domain name and stopped working recently.
- Several channel additions and updates.

### CODE REVIEW

File: channels/br.m3u [hunk 1]
```diff
@@ -67,34 +73,38 @@ https://evpp.mm.uol.com.br/ne10/ne10.smil/chunklist_w654628617_b216000_sleng.m3u
...
-http://69.46.0.170:1935/tvaparecida/tvaparecida.stream/live.m3u8?FORMIGATV
-#EXTINF:-1 tvg-id="TV Aparecida" tvg-name="TV Aparecida" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/xyYIdif.png" group-title="",TV Aparecida
-http://69.46.0.170:1935/tvaparecida/tvaparecida.stream/live.m3u8?TvFogoHD
-#EXTINF:-1 tvg-id="TV Aparecida" tvg-name="TV Aparecida" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/xyYIdif.png" group-title="",TV Aparecida
+#EXTINF:-1 tvg-id="TV Aparecida" tvg-name="TV Aparecida" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/xyYIdif.png" group-title="Religious",TV Aparecida
 http://caikron.com.br:1935/tvaparecida/tvaparecida.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/ejp65zr.png" group-title="",TV BNO
 http://tv02.logicahost.com.br:1935/bonner/bonner/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/dVLIKak.png" group-title="",TV Brasil 2
+https://22722d.ha.azioncdn.net/hls/STsEBCLIVE.m3u8
```
Comment (freearhey) [inline channels/br.m3u:81]: Link does not work

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by freearhey** [comment]
Please remove the broken link so we can approve this pull request.

File: channels/br.m3u (hunk 1 above)
Comment (rblopes) [reply channels/br.m3u:81]: Just tested here, and works fine for me.
I've been testing those links for a while.
Not sure if they are employing some kind of region blocking, though.

Comment (freearhey) [reply channels/br.m3u:81]: I just checked this link a couple more times with the VLC player and streamtest.in and it still doesn't work.
I guess you're right, it's just geo-blocked.
So it's really better to remove it.
Captura de pantalla 2020-04-13 a las 0 22 40

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thank you!

**Pull request merged**